### PR TITLE
Frontend HTTP helper function

### PR DESCRIPTION
**Pull request by David1000734**
# Pull Request Template

## Description
Added a helper function to the frontend that can handle various HTTP requests. Additionally, the links to the pictures for the app.json has been fixed and a new logo.png was added.

## Type of Change

- [x] Bugfix
- [x] New Feature
- [ ] Documentation Update
- [ ] Other (please describe):

## Checklist

- [ ] I have read the contributing guidelines.
- [x] I have followed the code style of this project.
- [ ] I have added tests to cover my changes.
- [x] I have updated the documentation (if applicable).
- [x] My changes do not generate new warnings.

## Related Issue
Addresses this issue however, while the server does not have a public address, it would not be possible to make a call from the frontend to the backend. #15 

## Additional Notes
In addition to the HTTP function, this pull will also add an .env file for the project that only contains the domain address for now.

### CODE REVIEW

File: Tabby/types/api_request.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    });
+
+    // HTTP DELETE Example
+    await http_callback({
+        domain: "https://reqres.in/api/",
+        route: 'users/2',
+        method: "DELETE",
+        type: "application/json"
+    });
+*/
```
Comment (LilAttacker) [inline Tabby/types/api_request.ts:93]: Why is this block commented out?

I don't really have an issue with it, but I would like to know if there is a reason on why.

**Review by LilAttacker — CHANGES_REQUESTED** [review]
Honestly this change looks incredible. Everything looks good and what is currently there I have no issue with.

That being said, I have noticed a distinct lack of any form of testing. I believe we currently only have a single front end test, and I think we need to improve that.

Please add at least a few tests.

**Comment by David1000734** [comment]
> Honestly this change looks incredible. Everything looks good and what is currently there I have no issue with.
> 
> That being said, I have noticed a distinct lack of any form of testing. I believe we currently only have a single front end test, and I think we need to improve that.
> 
> Please add at least a few tests.

I recognize the concern with testing however, the purpose of this push was to give Caesar and Josh a working HTTP function so that they can start having the frontend do calls to our backend. For this reason, can you allow me to push this change without the tests? I will have to come back to this at a later time figure out how to test functions in TypeScript

**Review by LilAttacker — APPROVED** [review]
I'm willing to overlook my concerns about the lack of testing with this if it means that further progress elsewhere can be done as a result.

**Pull request merged**